(DOCSP-27056): Flutter - Remove download progress notification docs (#2433)
mongodb/docs-realm: source/sdk/flutter/sync/manage-sync-session.txt

.. _flutter-monitor-sync-progress:

Monitor Sync Upload Progress
----------------------------

> To monitor Sync upload progress progress, call :flutter-sdk:`SyncSession.getProgressStream()

Reviewer: ```suggestion
To monitor Sync upload progress, call :flutter-sdk:`SyncSession.getProgressStream()
```

``SyncSession.getProgressStream()`` takes two arguments:

- A :flutter-sdk:`ProgressDirection <realm/ProgressDirection.html>`
  enum that must be set to ``upload``.

>   This specifies that the progress stream track uploads.

Reviewer 2: I don't think "track uploads" is grammatically correct here - we probably need to tweak the wording or say "tracks."

```suggestion
  This specifies that the progress stream tracks uploads.
```

> - A :flutter-sdk:`ProgressMode <realm/latest/realm/ProgressMode.html>` enum

Reviewer: ```suggestion
A :flutter-sdk:`ProgressMode <realm/ProgressMode.html>` enum
```

.. literalinclude:: /examples/generated/flutter/manage_sync_session_test.snippet.monitor-progress.dart
   :language: dart

.. warning:: Do Not Track Downloads

   The ``ProgressDirection`` enum also has a ``download`` option to track down downloads.

>    However, this option is not currently working. **Do not** use ``ProgressDirection.download``.

Reviewer 2: I wonder if there's a nicer way to say "this option is not currently working." Like, maybe:

> The ``download`` case provides planned future support for download progress notifications. However, these notifications do not currently provide an accurate indicator of download progress. Do not rely on ``ProgressDirection.download`` for download progress notifications.

We could probably wordsmith this more, but I think some of the discussion indicated it sometimes reported some number, so I'm not sure we can get away with just saying "this option doesn't work - don't use it." I think if devs do/have used it and it has reported numbers, they may ignore or argue with "this doesn't work" so it's probably better to be clear that it's not providing accurate info.

Author: thx! makes sense. will use your suggestion

>    The ``ProgressDirection`` enum also has a ``download`` option to track down downloads.
>    However, this option is not currently working. **Do not** use ``ProgressDirection.download``.

Author: ```suggestion
   The ``ProgressDirection`` enum also has a ``download`` option to track down downloads.
   The ``download`` case provides planned future support for download progress notifications. 
   However, these notifications do not currently provide an accurate indicator of download progress.
   Do not rely on ``ProgressDirection.download`` for download progress notifications.
```
